client/state/gravatar-status/test/reducer.js

```diff
@@ -9,37 +8,38 @@ import reducer, { isUploading, tempImage } from '../reducer';
 
 describe( 'reducer', () => {
 	test( 'exports expected reducer keys', () => {
-		expect( reducer( undefined, {} ) ).to.have.keys( [ 'isUploading', 'tempImage' ] );
+		expect( reducer( undefined, {} ) ).toHaveProperty( 'isUploading' );
+		expect( reducer( undefined, {} ) ).toHaveProperty( 'tempImage' );
```

Comment:
What about `expect( Object.keys( reducer() ) ).toEqual( [ 'isUploading', 'tempImage' ] )`?

Reply:
Interesting, but I think my version displays more transparently the intention of the test. I've improved it to call the reducer function just once in https://github.com/Automattic/wp-calypso/pull/62066/commits/30aae00303dfad9f235ae5141aef1f90eae5367e